torch/_higher_order_ops/triton_kernel_wrap.py

```diff
@@ -884,8 +884,12 @@ def call_run(self, variable, args, kwargs, tx):
         )
 
     def call_triton_kernel(self, variable, args, kwargs, tx):
+        from triton import JITFunction
         from triton.runtime.autotuner import autotune, Autotuner, Config
 
+        from torch._dynamo.variables.constant import ConstantVariable
+        from torch._dynamo.variables.tensor import SymNodeVariable
```

Comment:
@davidberard98 the code in this file isn't supposed to use Dynamo, it's a common codepath that works for both Dynamo and the non-strict export route (triton_op/capture_triton). This likely has introduced a bug on the triton_op/capture_triton side (if we try to rewrite the test case with triton_op/capture_triton). Can you look into fixing this?

Reply:
for future reference, should be fixed in #136686